packages/cli/src/commands/userDefined.mjs

```diff
@@ -0,0 +1,46 @@
+import path from 'node:path';
+import fs from 'fs';
+
+let userDefinedCommandsPaths = [];
+
+/**
+ * Extend the commands from a directory
+ * @param {string} commandsDir - The directory containing the commands
+ * @returns {Promise<void>}
+ */
+const extendCommandsFromDir = async (commandsDir) => {
+  userDefinedCommandsPaths = userDefinedCommandsPaths.concat(
+    fs
+      .readdirSync(commandsDir)
+      .map((command) => ({ dir: commandsDir, command }))
+      .filter(({ command, dir }) => {
+        return (
+          fs.statSync(path.join(dir, command)).isFile() &&
+          (command.endsWith('.js') ||
+            command.endsWith('.mjs') ||
+            command.endsWith('.cjs'))
+        );
+      }),
+  );
+};
+
+const files = fs.readdirSync(process.cwd());
```

Comment:
Merkur supports only merkur.config.mjs file so check can be simpler.

Reply:
Simplified to `const existsMerkurConfig = fs.readdirSync(process.cwd()).includes('merkur.config.mjs');`

Reply:
Updated to
```
const existsMerkurConfig = fs
  .readdirSync(process.cwd())
  .includes('merkur.config.mjs');
```